HHS/Head-Start-TTADP
# [TTAHUB-727] Cleanup Admin Page and Add 'National Center' user role

## Description of change

Cleanup the spacing between the global permissions.

Add three new user roles: 

- National Center
- Customer Service Contract
- Other Federal Staff

## How to test

- Go to the user page. Verify the global permission check boxes are now spaced. 
- Verify the new user roles exist in specified order.
- Verify the new user roles can be saved.
- Create a report with a user that has the new roles.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-727


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
